modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -723,6 +743,15 @@ class CV_EXPORTS_W QRCodeDetector
     */
 CV_EXPORTS bool detectQRCode(InputArray in, std::vector<Point> &points, double eps_x = 0.2, double eps_y = 0.1);
 
+/** @brief Detect QR codes in image and return vector of minimum area of quadrangle that describes QR codes.
+    @param in  Matrix of the type CV_8UC1 containing an image where QR codes are detected.
+    @param points Output vector of vector of vertices of a quadrangle of minimal area that describes QR codes.
+    @param eps_x Epsilon neighborhood, which allows you to determine the horizontal pattern of the scheme 1:1:3:1:1 according to QR code standard.
+    @param eps_y Epsilon neighborhood, which allows you to determine the vertical pattern of the scheme 1:1:3:1:1 according to QR code standard.
+    */
+CV_EXPORTS bool multipleDetectQRCode(InputArray in, std::vector< std::vector< Point > > &points,
+                                     double eps_x = 0.2, double eps_y = 0.1);
```

Comment:
`detectQRCodeMulti` + parameters

Reply:
Do we really need two different epsilons? I mean according to QR codes standard, both horizontal and vertical directions should be equal, aren't they?